FairBid/unityads.plist.xml

```diff
@@ -79,11 +79,6 @@
                 <key>SKAdNetworkIdentifier</key>
                 <string>f73kdq92p3.skadnetwork</string>
             </dict>
-            <dict>
```

Comment:
this I want remove. I mean it depends whether unityads also deprecated it. Otherwise the SkAdNetworkReportGenerator DAG will always send a report saying that we have a missing identifier.
This DAG is checking difference between this repo and the network skadnetwork data

Reply:
I put back the deleted entry :)